packages/libs/error-stack/src/fmt/hook.rs

```diff
@@ -585,6 +585,8 @@ impl<T: 'static> HookContext<T> {
     pub fn increment_counter(&mut self) -> isize {
         let counter = self.get_mut::<Counter>();
 
+        // reason: This would fail as we cannot move out of `self` because it is borrowed
+        #[allow(clippy::option_if_let_else)]
```

Comment:
I wish we could use expect 🙃

Reply:
Me too. As soon as https://github.com/rust-lang/rust/issues/54503 is stabilized 🙂